migrations/20201021191942_fs_type_field.sql

```diff
@@ -0,0 +1,4 @@
+-- Add migration script here
+CREATE TYPE fs_type AS ENUM('zfs', 'ldiskfs');
+
+ALTER TABLE target add column fs_type fs_type;
```

Comment:
```suggestion
ALTER TABLE IF EXISTS target add column fs_type fs_type;
```

Reply:
I didn't add this initially because I thought the timestamp on the migration files were ran in order. But maybe this could cause an issue with upgrading? In any case, I'll add it in.

Reply:
> I didn't add this initially because I thought the timestamp on the migration files were ran in order.

They are.

> But maybe this could cause an issue with upgrading?

What issue are you thinking of?

Reply:
I was trying to think of what problem this would cause if we leave out the `IF EXISTS`. I figured since the migrations are run in order that we could be certain that the `target` table would be there.

Reply:
We can't be certain because people may partially clear out tables during development. This sort of helps out with allowing things to move forward in that case, but it doesn't really matter in production, as we'd get an error in either case.

Reply:
Thanks, i've added it.